customer_satisfaction/README.md

```diff
@@ -40,21 +38,23 @@ Our standard training pipeline consists of several steps:
 * model_train  :- This step will train the model and will save the model using mlfow autlogging. 
 * evaluation   :- This step will evaluate the model and will save the metrics using mlfow autlogging into the artifact store.  
 
-We have another pipeline which is the deployment_pipeline.py that implements a continuous deployment workflow. It ingests and processes input data, trains a model and then (re)deploys the prediction server that serves the model if it meets some evaluation criteria.
+### Depoloyment & Inference Pipeline 
 
-We also have an inference pipeline that interacts with the prediction server deployed by the continuous deployment pipeline to get online predictions based on live data. 
+We have another pipeline which is the deployment_pipeline.py that implements a continuous deployment workflow. It ingests and processes input data, trains a model and then (re)deploys the prediction server that serves the model if it meets some evaluation criteria.
 
 In the deployment pipeline, ZenML's MLflow tracking integration is used to log the hyperparameter values -- as well as the trained model itself and the model evaluation metrics -- as MLflow experiment tracking artifacts into the local MLflow backend. This pipeline also launches a local MLflow deployment server to serve the latest MLflow model if its accuracy is above a configured threshold.
 
 The MLflow deployment server is running locally as a daemon process that will continue to run in the background after the example execution is complete.
 
 The deployment pipeline has caching enabled to avoid re-training the model if the training data and hyperparameter values don't change. When a new model is trained that passes the accuracy threshold validation, the pipeline automatically updates the currently running MLflow deployment server so that the new model is being served instead of the old one.
 
-The inference pipeline simulates loading data from a dynamic external source, then uses that data to perform online predictions using the running MLflow prediction server.
+We also have an inference pipeline that interacts with the prediction server deployed by the continuous deployment pipeline to get online predictions based on live data. The inference pipeline simulates loading data from a dynamic external source, then uses that data to perform online predictions using the running MLflow prediction server.
+
+![training_and_deployment_pipeline](_assets/training_and_deployment_pipeline.png)
```

Comment:
could you make the line connecting the training pipeline at the top down to where it is in the bigger pipeline below a bit bigger / bolder? Also maybe make the captions for each pipeline ("Training Pipeline" etc) a bit bigger? They're really small right now.